ljh/ljh.go

```diff
@@ -208,10 +208,11 @@ func (w Writer) Close() {
 // rowcount is framecount*number_of_rows+row_number for TDM or TDM-like (eg CDM) data,
 // rowcount=framecount for uMux data
 // return error if data is wrong length (w.Samples is correct length)
-func (w *Writer) WriteRecord(rowcount int64, timestamp int64, data []uint16) error {
+func (w *Writer) WriteRecord(framecount int64, timestamp int64, data []uint16) error {
 	if len(data) != w.Samples {
 		return fmt.Errorf("ljh incorrect number of samples, have %v, want %v", len(data), w.Samples)
 	}
+	rowcount := framecount*int64(w.NumberOfRows) + int64(w.RowNum)
```

Comment:
Does this mean that LJH 2.1 will contain row-count and 2.2 or 3.0 will contain frame-count?

I think I can live with that.